src/components/button/button.api.json

```diff
@@ -297,19 +284,7 @@
                 ]
               },
               {
-                "title": "Inverted",
```

Comment:
I feel like you can leave the `Inverted`.
If you do return it then there's an `Inverted` section under `Icon Button` that was removed.
If not, there's a section below about `Inverted` that's still there.